Update automatic beacon algorithm to send to all registered URLs

blu25 opened the pull request:
Automatic beacons will now send to all URLs that registered with the reserved.top_navigation event type, regardless of what data was set when calling setReportEventDataForAutomaticBeacons(). However, automatic beacon data will only be included in the beacon if the beacon's destination is specified in the destination field when calling setReportEventDataForAutomaticBeacons().


<!--
    This comment and the below content is programmatically generated.
    You may add a comma-separated list of anchors you'd like a
    direct link to below (e.g. #idl-serializers, #idl-sequence):

    Don't remove this comment or modify anything below this line.
    If you don't want a preview generated for this pull request,
    just replace the whole of this comment's content by "no preview"
    and remove what's below.
-->
***
<a href="https://pr-preview.s3.amazonaws.com/WICG/fenced-frame/pull/122.html" title="Last updated on Sep 22, 2023, 3:15 PM UTC (d598d24)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WICG/fenced-frame/122/bd99c82...d598d24.html" title="Last updated on Sep 22, 2023, 3:15 PM UTC (d598d24)">Diff</a>

domfarolino commented:
Do the existing WPTs assert this behavior, or have we added any more that we could now pull into a `<wpt>` block?

blu25 commented:
> Do the existing WPTs assert this behavior, or have we added any more that we could now pull into a `<wpt>` block?

There's one more that was just merged upstream. Added it to the list of automatic beacon WPTs.
